atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellLoader.java

```diff
@@ -140,8 +140,11 @@ void loadWithTs(String kvsMethodName,
             final byte[] col = entry.getKey();
             Collection<Cell> columnCells = entry.getValue();
             if (columnCells.size() > fetchBatchCount) {
```

Comment:
Should this even be configured by this config? I'd err on the side of making it constant. Also make sure it is consistent with the `TS_PER_BATCH` below

Reply:
We don't have visibility of `CassandraConstants` here. I can shove this into `AtlasDbConstants` maybe - will have a look at doing that